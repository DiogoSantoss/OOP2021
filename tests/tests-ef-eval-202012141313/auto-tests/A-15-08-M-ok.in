8
2
S2
1
6M2
1
2
R2
1
6M2
3
2
R2
1
3O3
2
2
R2
1
2H2
2
2
R2
1
7M5
1
4
4
4
2
4
1
4
0
0
9
2
R2
2
S2
0
8
4
3
0
9
2
R2
2
S2
0
0
