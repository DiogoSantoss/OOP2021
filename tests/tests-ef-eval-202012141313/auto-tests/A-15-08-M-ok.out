Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Identificador do cliente: 1|R2|6M2|3|12|12|1|0
2|R2|3O3|2|16|16|1|0
4|R2|7M5|1|8|8|1|0
Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Identificador do cliente: 0|S2|6M2|1|4|4|1|0
Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Identificador do cliente: 1|R2|6M2|3|12|12|1|0
2|R2|3O3|2|16|16|1|0
3|R2|2H2|2|8|8|1|0
4|R2|7M5|1|8|8|1|0
Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Identificador do cliente: 0|S2|6M2|1|4|4|1|0
Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 